src/wasm-type.h

```diff
@@ -137,28 +142,32 @@ class Type {
   Type(Rtt);
 
   // Predicates
-  //                 Compound Concrete
-  //   Type        Basic │ Single│
-  // ╒═════════════╦═│═╤═│═╤═│═╤═│═╤═══════╕
-  // │ none        ║ x │   │   │   │       │
-  // │ unreachable ║ x │   │   │   │       │
-  // ├─────────────╫───┼───┼───┼───┤───────┤
-  // │ i32         ║ x │   │ x │ x │ I     │ ┐ Number
-  // │ i64         ║ x │   │ x │ x │ I     │ │  I_nteger
-  // │ f32         ║ x │   │ x │ x │   F   │ │  F_loat
-  // │ f64         ║ x │   │ x │ x │   F   │ │  V_ector
-  // │ v128        ║ x │   │ x │ x │     V │ ┘
-  // ├─ Aliases ───╫───┼───┼───┼───┤───────┤
-  // │ funcref     ║ x │   │ x │ x │ f  n  │ ┐ Ref
-  // │ anyref      ║ x │   │ x │ x │ f? n  │ │  f_unc
-  // │ eqref       ║ x │   │ x │ x │    n  │ │  n_ullable
-  // │ i31ref      ║ x │   │ x │ x │       │ │
-  // │ dataref     ║ x │   │ x │ x │       │ │
-  // ├─ Compound ──╫───┼───┼───┼───┤───────┤ │
-  // │ Ref         ║   │ x │ x │ x │ f? n? │◄┘
-  // │ Tuple       ║   │ x │   │ x │       │
-  // │ Rtt         ║   │ x │ x │ x │       │
-  // └─────────────╨───┴───┴───┴───┴───────┘
+  //                      Compound Concrete
+  //   Type             Basic │ Single│
+  // ╒══════════════════╦═│═╤═│═╤═│═╤═│═╤═══════╕
+  // │ none             ║ x │   │   │   │       │
+  // │ unreachable      ║ x │   │   │   │       │
+  // ├──────────────────╫───┼───┼───┼───┤───────┤
+  // │ i32              ║ x │   │ x │ x │ I     │ ┐ Number
+  // │ i64              ║ x │   │ x │ x │ I     │ │  I_nteger
+  // │ f32              ║ x │   │ x │ x │   F   │ │  F_loat
+  // │ f64              ║ x │   │ x │ x │   F   │ │  V_ector
+  // │ v128             ║ x │   │ x │ x │     V │ ┘
+  // ├─ Aliases ────────╫───┼───┼───┼───┤───────┤
+  // │ funcref          ║ x │   │ x │ x │ f  n  │ ┐ Ref
+  // │ anyref           ║ x │   │ x │ x │ f? n  │ │  f_unc
+  // │ eqref            ║ x │   │ x │ x │    n  │ │  n_ullable
+  // │ i31ref           ║ x │   │ x │ x │       │ │
+  // │ dataref          ║ x │   │ x │ x │       │ │
+  // │ stringref        ║ x │   │ x │ x │    n  │ │
+  // │ stringview_wtf8  ║ x │   │ x │ x │    n  │ │
+  // │ stringview_wtf16 ║ x │   │ x │ x │    n  │ │
+  // │ stringview_iter  ║ x │   │ x │ x │    n  │ │
```

Comment:
So all `stringref` and `stringview`s are nullable? ([The spec](https://github.com/WebAssembly/stringref/blob/main/proposals/stringref/Overview.md) says `stringref` is, but no mention of `stringview`s.

Reply:
Yes, I can't find that in the spec either. I think nullable is more likely, but we can adjust later. The spec is still very early draft.

Reply:
We are going to transition all the shorthands to be nullable, so this is a safe assumption.